Automattic/pocket-casts-ios
# Tracks: Add analytics to the Auto Archive settings

| 📘 Project: #154 | 🛫 Depends on: #308  |
|:---:|:---:|

- `settings_auto_archive_shown`: When the Auto Archive setting is shown to the user
- `settings_auto_archive_played_changed`: When the user changes the Auto Archive option
- `settings_auto_archive_inactive_changed`: When the user changes the Archive Inactive Episodes option
- `settings_auto_archive_include_starred_toggled`: When the user toggles the Include Starred episodes option


## To test

1. Go to the profile tab
2. Tap on the cog
3. Tap on Auto Archive
4. ✅ `🔵 Tracked: settings_auto_archive_shown`
9. Go through each setting and change its value
12. Check console and verify each setting has a corresponding event in console and the value displayed is the value that you selected
13. If the option changed is a switch then the event should be suffixed `_toggle` if it's a value change it should be suffixed with `_changed`


## Checklist
